### PR TITLE
refactor(tests): Invert asserts in unit test #4197

### PR DESCRIPTION
**Pull request by GonzaloSaad**
## Issue
#4197

## Work done
- [x] Flip asserts to comply with `(expected, actual)`
- [x] Change `assertEquals(<Constant>, obj)` to `assert<Constant>(obj)`
- [x] Minor style refactors on multiline statements

### CODE REVIEW

**Comment by GonzaloSaad** [comment]
I found this assertion here that confused me haha: 
https://github.com/localstack/localstack/blob/4d3cfd1bd3abf85b20653e72575af928f5c95e53/tests/unit/test_s3.py#L320

I wanted to rewrite it, but could not. Basically is always going to pass regarless the condition. 

```
>>> True >= False
True
>>> False >= False
True
```

**Comment by whummer** [comment]
> I found this assertion here that confused me haha:

Good catch! :) I think we can rewrite as:
```
self.assertNotIn(bucket, bucket.instances or []) 
```
Same for line 307:
```
self.assertNotIn(key, key.instances or [])
```

File: tests/integration/serverless/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "version": "serverless --version"
   },
   "devDependencies": {
-    "serverless": "^2.35.0",
+    "serverless": "2.48.0",
```
Comment (GonzaloSaad) [inline tests/integration/serverless/package.json:11]: I do not know what I did wrong when pulling from master. 
I'm used to run:
`git pull origin master --rebase` (I used to perform merge commits before)
`git push`
:tada: 

It is the first time the commits already merged in master show in the diff! Any advice/tip/hint it is appreaciated!

Comment (whummer) [reply tests/integration/serverless/package.json:11]: I guess that should be fine when we squash-merge the PR - no worries.

What you have above should usually work. To be on the safe side, you could also try these steps manually:
```
git checkout master
git pull
git checkout your-branch
git rebase master
git push origin your-branch
```

---
Anyway. Thanks for the PR @GonzaloSaad ! 👍 - should be good to merge.

**Pull request merged**